README.md

```diff
@@ -47,17 +47,18 @@ For a manual self-hosted production installation these are the recommended steps
 1. Setup the database with `yarn db:migrate`. Production assumes an SSL connection to the database by default, if
    Postgres is on the same machine and is not SSL you can migrate with `yarn db:migrate --env=production-ssl-disabled`, for example:
 
-   `docker run --rm outlinewiki/outline yarn db:migrate`
+   `docker run --rm --env-file=.env outlinewiki/outline yarn db:migrate`
```

Comment:
Would prefer to revert these lines, it makes the assumption that your environment variables are stored in a file. We describe this in the first step.

Reply:
Recently I was confused about 5 minutes why command doesn't work. Copy paste instruction is what we need. For now I need to pass `--env` parameter to every docker command, is it right?

Reply:
Well, _it depends_ – what's copy and paste for your setup is not for another. Many environments will inject environment variables through other mechanisms

Reply:
Ok, I'll remove that. Thank you. Please share your thoughts how to pass env variables in another way. Maybe my workaround not so perfect for production mode.